### PR TITLE
Decrease stress test traffic when using op contents.

### PR DESCRIPTION
**Pull request by andre4i**
## Description

Large ops (or ops with content in general) in the stress tests are problematic as they risk creating huge files and cause unnecessary increase in COGS for the server, leading to a cascade of random errors related to fetching snapshots and polluting the alerts. 

This change re-enables large ops, however it adds the following limits:

- only 12 runners can send large ops
- there will be at most 5 large ops of at exactly 214800 bytes each, per run per enabled runner

therefore, **the max file size created for each test run would be 12 x 5 x 214800 = 12.8 MB**. And the **max snapshot size would be 12 x 214800 = 2.5 MB** (as each runner will reuse its own runId as a key in the map where its ops are going, overwriting existing content)

### CODE REVIEW

File: packages/test/test-service-load/src/loadTestDataStore.ts [hunk 1]
```diff
@@ -574,10 +574,23 @@ class LoadTestDataStore extends DataObject implements ILoadTest {
...
+				this.shouldSendLargeOp(
+					opSizeinBytes,
+					largeOpRate,
+					opsSent,
+					largeOpJitter,
+					config.runId,
+					maxClientsSendingLargeOps,
+				)
+			) {
 				dataModel.sharedmap.set(`key${opsSent}`, generateContentOfSize(getOpSizeInBytes()));
```
Comment (anthony-murphy) [inline packages/test/test-service-load/src/loadTestDataStore.ts:594]: is this key still the problem. i don't know why we would ever need more than a single key. at most i think a key per runner or runId

Comment (andre4i) [reply packages/test/test-service-load/src/loadTestDataStore.ts:594]: Sure, but I don't believe this to be a problem (at least not right now). There will be 10 map keys created for a run. Is that problematic?

**Comment by andre4i** [comment]
Currently tested in a public branch at https://github.com/microsoft/FluidFramework/tree/test/andreiiacob

File: packages/test/test-service-load/src/loadTestDataStore.ts (hunk 1 above)
Comment (anthony-murphy) [reply packages/test/test-service-load/src/loadTestDataStore.ts]: i don't know. its hard for me to understand the interplay of all the settings, and if there is a bad config matrix, as ideally any config works, it just does more or less. we are not there today, but the fewer traps lying around the better

Comment (anthony-murphy) [reply packages/test/test-service-load/src/loadTestDataStore.ts]: to me this feature isn't trying to grow the snapshot, so it should avoid doing it more than it needs to

**Comment by andre4i** [comment]
This change has successfully been tested in a private branch.

Error results:

Data_branch | Data_hostName | runId | sum_count_ | min_min_Event_Time
-- | -- | -- | -- | --
refs/heads/test/andreiiacob | @fluid-internal/test-service-load | 130495-routerlicious-frs-ci_frs | 81 | 2023-02-14 21:53:46.6330000
refs/heads/test/andreiiacob | @fluid-internal/test-service-load | 130495-tinylicious--ci | 30 | 2023-02-14 21:56:09.4040000
refs/heads/test/andreiiacob | @fluid-internal/test-service-load | 130495-odsp-odsp-df-ci | 17 | 2023-02-14 21:57:57.8740000
refs/heads/test/andreiiacob | @fluid-internal/test-service-load | 130495-odsp--ci | 20 | 2023-02-14 22:06:08.0480000


</BODY></HTML>

Lots of which were a [canary error log](https://github.com/microsoft/FluidFramework/commit/01c5cd7f3c9654573dd364ca4c5c872fd23b4770) added by me:

Data_eventName | count_
-- | --
fluid:telemetry:Summarizer:summarizingError | 28
fluid:telemetry:Summarizer:Running:Summarize_cancel | 28
fluid:telemetry:OpPerf:OpRoundtripTime | 8
fluid:telemetry:DeltaManager:NoJoinOp | 25
fluid:telemetry:DeltaManager:ReceivedJoinOp | 25
LargeTestPayload | 31
fluid:telemetry:Summarizer:Running:IncrementalSummaryViolation | 1
fluid:telemetry:OdspDriver:DeltaConnection:FlushResult | 2


</BODY></HTML>

Times the stress tests did chunking:

Data_buildId | Data_driverType | Data_driverEndpointName | Data_profile | count_ | Data_branch
-- | -- | -- | -- | -- | --
130495 | odsp | odsp-df | ci | 11 | refs/heads/test/andreiiacob
130495 | odsp |   | ci | 10 | refs/heads/test/andreiiacob
130495 | tinylicious |   | ci | 4 | refs/heads/test/andreiiacob
130495 | routerlicious | frs | ci_frs | 7 | refs/heads/test/andreiiacob


</BODY></HTML>

Detailed metrics about each instance chunking happened (across all drivers, all test runs for this build):

Event_Time | Data_length | Data_sizeInBytes | Data_chunks | Data_chunkSizeInBytes
-- | -- | -- | -- | --
2023-02-14 21:57:24.7110000 | 1 | 1305780 | 3 | 614400
2023-02-14 21:57:57.8830000 | 3 | 1305696 | 5 | 307200
2023-02-14 22:00:03.1090000 | 1 | 1305744 | 2 | 706800
2023-02-14 22:04:16.0960000 | 1 | 1305732 | 3 | 614400
2023-02-14 22:04:48.7380000 | 3 | 1306036 | 3 | 614400
2023-02-14 22:06:08.0570000 | 3 | 1306292 | 3 | 614400
2023-02-14 22:06:17.1300000 | 1 | 1305736 | 3 | 614400
2023-02-14 22:07:11.4650000 | 1 | 1305948 | 3 | 614400
2023-02-14 22:08:23.1620000 | 1 | 1306060 | 2 | 706800
2023-02-14 22:08:47.9730000 | 1 | 1305864 | 5 | 307200
2023-02-14 22:09:18.4470000 | 1 | 1306224 | 3 | 614400
2023-02-14 22:10:24.9580000 | 1 | 1306400 | 3 | 614400
2023-02-14 22:10:51.9070000 | 1 | 1305684 | 2 | 706800
2023-02-14 22:12:13.8650000 | 1 | 1305736 | 3 | 614400
2023-02-14 22:14:23.5070000 | 1 | 1306144 | 2 | 706800
2023-02-14 22:15:01.5400000 | 1 | 1305532 | 2 | 706800
2023-02-14 22:19:55.5530000 | 1 | 1306376 | 3 | 614400
2023-02-14 22:21:07.7920000 | 1 | 1306268 | 3 | 614400
2023-02-14 22:21:29.9910000 | 1 | 1305664 | 3 | 614400
2023-02-14 22:22:32.2360000 | 1 | 1306084 | 5 | 307200
2023-02-14 22:24:48.5000000 | 1 | 1305824 | 3 | 614400
2023-02-14 22:25:18.4250000 | 1 | 1306440 | 3 | 614400
2023-02-14 22:28:33.4350000 | 1 | 1305944 | 3 | 614400
2023-02-14 22:30:03.6690000 | 1 | 1305932 | 5 | 307200
2023-02-14 22:32:41.9970000 | 3 | 1306212 | 2 | 706800
2023-02-14 22:33:08.6780000 | 1 | 1306048 | 2 | 706800
2023-02-14 22:34:05.1950000 | 1 | 1305584 | 2 | 706800
2023-02-14 22:34:15.8410000 | 3 | 1306172 | 2 | 706800
2023-02-14 22:34:18.6330000 | 3 | 1306148 | 2 | 706800
2023-02-14 22:36:50.7520000 | 3 | 1306364 | 5 | 307200
2023-02-14 22:38:45.0460000 | 1 | 1306116 | 3 | 614400
2023-02-14 22:38:50.2240000 | 3 | 1306116 | 5 | 307200


</BODY></HTML>

`Data_sizeInBytes` is the size of the payload _after_ compression.

**Comment by vladsud** [comment]
I'd try to make it not limited to number of clients, but rather find other ways to make it proportional to payload.
ODSP team runs stress tests that hit 10x+ scale of ours (leveraging many clients machines), and it would be great to see them generating more of the large ops (and stress test PUSH by doing so), and not be limited to same workload that our single-machine runs expose.

**Comment by andre4i** [comment]
Will open a new PR with the changes from https://github.com/microsoft/FluidFramework/tree/test/andreiiacob after testing is complete.

**Pull request closed without merging**